tests/integration/standard/test_application_info.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2025 ScyllaDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from cassandra.application_info import ApplicationInfo
+from tests.integration import TestCluster, use_single_node, remove_cluster, xfail_scylla
+
+
+def setup_module():
+    use_single_node()
+
+
+def teardown_module():
+    remove_cluster()
+
+
+@xfail_scylla
+class ApplicationInfoTest(unittest.TestCase):
+    attribute_to_startup_key = {
```

Comment:
That's a wrong usage, the test most likely does not run now. xfail_scylla is defined as:\
`xfail_scylla = lambda reason, *args, **kwargs: pytest.mark.xfail(SCYLLA_VERSION is not None, reason=reason, *args, **kwargs)`, so you need to use it like `@xfail_scylla('Scylla does not have client_info column in system.clients / system_views.clients')`